components/Cards/SchoolCard/SchoolCard.js

```diff
@@ -23,40 +27,101 @@ class SchoolCard extends Component {
     logoSource: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
     website: PropTypes.string.isRequired,
+    toggleModal: PropTypes.func.isRequired,
   };
 
   static defaultProps = {
-    hasHousing: false,
+    hasHousing: true,
   };
 
-  state = {
-    isFrontOfCardShowing: true,
+  toggleModalClick = () => {
+    const { name, locations, toggleModal } = this.props;
+    toggleModal({ name, locations });
   };
 
-  showBackOfCard = () => this.setState({ isFrontOfCardShowing: false });
+  render() {
+    const { props } = this;
 
-  showFrontOfCard = () => this.setState({ isFrontOfCardShowing: true });
+    const hasManyLocations = props.locations.length > 1;
 
-  render() {
-    const { props, state } = this;
+    let locationText = 'Multiple locations';
+    if (props.hasOnlyOnline) {
+      locationText = 'Online only';
+    } else if (!hasManyLocations) {
+      const [location] = props.locations;
+      locationText = `${location.city}, ${location.state}`;
+    }
+
+    const analyticsObject = {
+      action: 'Button Selected',
+      category: 'Interactions',
+      label: `${props.name} | Locations`,
+    };
 
     return (
-      <Card
-        className={classNames(styles.SchoolCard, {
-          [styles.backCard]: !state.isFrontOfCardShowing,
-        })}
-        hasAnimationOnHover={false}
-      >
-        {state.isFrontOfCardShowing ? (
-          <FrontSchoolCard {...props} cardFlipCallback={this.showBackOfCard} />
-        ) : (
-          <BackSchoolCard
-            cardFlipCallback={this.showFrontOfCard}
-            locations={props.locations}
-            logoSource={props.logoSource}
-            schoolName={props.name}
-          />
-        )}
+      <Card className={styles.SchoolCard} hasAnimationOnHover={false}>
+        <>
+          <div className={styles.giBillRibbon}>GI Bill</div>
```

Comment:
This should be conditionally rendered based upon whether or not any one location _has_ the GI Bill

Reply:
This was total mind blank on my part. I read that in criteria and just forgot to make it conditionally render. My bad!